llvm/include/llvm/SYCLLowerIR/ModuleSplitter.h

```diff
@@ -225,6 +226,9 @@ class ModuleDesc {
 
   void saveSplitInformationAsMetadata();
 
+  ModuleDesc makeDummy() const;
```

Comment:
```suggestion
  ModuleDesc makeDummyCopy() const;
```

To better indicate that it creates a clone. Otherwise the name may suggest that it simply modifies the existing `ModuleDesc`